test/test-cases/regression/variable-MULTIPART_UNMATCHED_BOUNDARY.json

```diff
@@ -57,5 +57,206 @@
       "SecRuleEngine On",
       "SecRule MULTIPART_UNMATCHED_BOUNDARY \"@contains small_text_file.txt\" \"id:1,phase:3,pass,t:trim\""
     ]
+  },
+  {
+    "enabled":1,
+    "version_min":300000,
+    "title":"Testing Variables :: MULTIPART_UNMATCHED_BOUNDARY - 2nd unmatched",
+    "client":{
+      "ip":"200.249.12.31",
+      "port":123
+    },
+    "server":{
+      "ip":"200.249.12.31",
+      "port":80
+    },
+    "request":{
+      "headers":{
+        "Host":"localhost",
+        "User-Agent":"curl/7.38.0",
+        "Accept":"*/*",
+        "Content-Length":"330",
+        "Content-Type":"multipart/form-data; boundary=--------------------------756b6d74fa1a8ee2",
+        "Expect":"100-continue"
+      },
+      "uri":"/",
+      "method":"POST",
+      "body":[
+        "----------------------------756b6d74fa1a8ee2",
+        "Content-Disposition: form-data; name=\"a\"",
+        "",
+        "1",
+        "----------------------------wrong-unmatched",
+        "Content-Disposition: form-data; name=\"b\"",
+        "",
+        "2",
+        "----------------------------756b6d74fa1a8ee2",
+        "Content-Disposition: form-data; name=\"c\"",
+        "",
+        "3",
+        "----------------------------756b6d74fa1a8ee2",
+        "Content-Disposition: form-data; name=\"d\"",
+        "",
+        "4",
+        "----------------------------756b6d74fa1a8ee2",
+        "Content-Disposition: form-data; name=\"e\"",
+        "",
+        "5",
+        "----------------------------756b6d74fa1a8ee2--",
+        ""
+      ]
+    },
+    "response":{
+      "headers":{
+        "Date":"Mon, 13 Jul 2015 20:02:41 GMT",
+        "Last-Modified":"Sun, 26 Oct 2014 22:33:37 GMT",
+        "Content-Type":"text/html"
+      },
+      "body":[
+        "no need."
+      ]
+    },
+    "expected":{
+      "debug_log":"Target value: \"1\" \\(Variable: MULTIPART_UNMATCHED_BOUNDARY\\)",
+      "http_code": 403
+    },
+    "rules":[
+      "SecRuleEngine On",
+      "SecRule MULTIPART_UNMATCHED_BOUNDARY \"@eq 1\" \"id:1,phase:2,deny,status:403\""
```

Comment:
You expect value `1` but the PR #1747 has modified the logic, that in this case the engine gives `2`. See the detailed information in #1801.